Import-Encord-Active-Model-Predictions/Micromodels-generate-detection-predictions.ipynb

```diff
@@ -159,11 +266,51 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "os.chdir(ea_project_fs.project_dir.as_posix()) \n",
+    "# Change the current working directory to the project directory\n",
+    "os.chdir(ea_project_fs.project_dir.as_posix())\n",
+    "\n",
+    "# Import the predictions file into Encord Active\n",
     "! encord-active import predictions {prediction_file}\n",
     "\n",
     "print(\"\\n✅︎ Predictions are imported successfully!\")"
    ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c90ddae8",
+   "metadata": {},
+   "source": [
+    "# ✅ Wrap up"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ee272d7a",
+   "metadata": {},
+   "source": [
+    "\n",
+    "📓This Colab notebook showed you how to generate detection boxes with 🟣 Encord Active using Micro-Models in Encord Annotate.\n",
+    "\n",
+    "---\n",
+    "\n",
+    "🟣 Encord Active is an open-source framework for computer vision model testing, evaluation, and validation. Check out the project on [GitHub](https://github.com/encord-team/encord-active), leave a star 🌟 if you like it, and leave an issue if you find something is missing.\n",
```

Comment:
same purple circle here

Reply:
Thanks a lot, Gorkem. The reason was more akin to "branding". It makes it easy to identify and users would be able to map the relationship after going through a few notebooks.